project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.7.3
```

Comment:
Sorry, I'm trying to keep the 0.4.x series pinned to sbt 1.6 :)

Reply:
Ok, that's fine. I'll drop that commit, it just makes it hard to confirm that the issue is resolved within the project itself.